RFS/src/main/java/com/rfs/version_es_6_8/ElasticsearchConstants_ES_6_8.java

```diff
@@ -1,11 +1,16 @@
 package com.rfs.version_es_6_8;
 
+import com.fasterxml.jackson.core.JsonFactory;
 import com.fasterxml.jackson.core.JsonGenerator;
 import com.fasterxml.jackson.core.JsonParser;
 import com.fasterxml.jackson.dataformat.smile.SmileFactory;
 import com.fasterxml.jackson.dataformat.smile.SmileGenerator;
 
 public class ElasticsearchConstants_ES_6_8 {
+    private ElasticsearchConstants_ES_6_8() {
+        throw new UnsupportedOperationException("This class is not meant to be instantiated");
```

Comment:
Do you need this?  I'm mostly curious because I've usually just done empty bodies for private constructors that aren't meant to be instantiated, figuring that it was up to the author to know if a class's own implementation did or didn't need to be instantiated.